.github/workflows/ci.yml

```diff
@@ -67,9 +67,16 @@ jobs:
       matrix:
         platform: [win32, x64, arm64]
     env:
-      CONFIGURATION: Release
       PLATFORM: ${{ matrix.platform }}
     steps:
+      - name: Setup environment
+        run: |
+          BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)
+          if [[ $BRANCH == 'develop' ]]; then
+             echo "CONFIGURATION=ReleaseLTCG" >> "$GITHUB_ENV"
+          else
+             echo "CONFIGURATION=Release" >> "$GITHUB_ENV"
```

Comment:
Shouldn't this be inverted? Pick the "Release" configuration on develop, and "ReleaseLTCG" otherwise (PRs). Checking for "master" should also be added. Or does `$BRANCH` hold the name of the branch it merges to?

Reply:
Never mind, I now see that the Release configuration is changed to not have whole program optimization. The configuration names could be clearer. What about "Release" and "ReleaseOptimized"?